src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -285,6 +285,97 @@ describe('<IncidentForm />', () => {
 
         expect(nextSpy).toHaveBeenCalledTimes(2)
       })
+
+      it('should postpone submit when postponeSubmitWhenLoading is defined for loadingQuestions', () => {
```

Comment:
I don't understand what this means. Does it mean "it postpones submit in case questions need to be fetched from the backend but haven't loaded yet"?

Reply:
It means it should postpone in case `loadingQuestions` is present in `postponeSubmitWhenLoading`.